src/components/MapView/Legends/exposedPopulationAnalysis.tsx

```diff
@@ -80,20 +83,49 @@ const ExposedPopulationAnalysis = ({
     await dispatch(requestAndStoreExposedPopulation(params));
   };
 
-  const ResultSwitches = () => (
-    <FormGroup>
-      <AnalysisFormControlLabel
-        control={
-          <Switch
-            color="primary"
-            checked={isMapLayerActive}
-            onChange={e => dispatch(setIsMapLayerActive(e.target.checked))}
+  const ResultSwitches = () => {
+    const data = useSelector(analysisResultSelector);
+    const features = data?.featureCollection.features;
+    const hasData = features?.length === 0;
+
+    const handleTableViewChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+      dispatch(setIsDataTableDrawerActive(e.target.checked));
+      dispatch(
+        setCurrentDataDefinition({
+          id: TableDefinitions.animal_deaths.id,
```

Comment:
@erwagasore why is referencing `animal_deaths` which is a country-specific dataset? Not sure what we are trying to achieve here?

Reply:
Looking into it...

Reply:
@ericboucher just created a fix https://github.com/WFP-VAM/prism-frontend/pull/373